src/samplers/hmc.jl

```diff
@@ -47,7 +47,7 @@ function HMC(alg::HMC, new_gid::Int)
 end
 HMC{T}(alg::HMC, new_gid::Int) where {T} = HMC(alg, new_gid)
 
-function hmc_step(θ, lj, lj_func, grad_func, H_func, ϵ, alg::HMC, momentum_sampler::Function;
+function hmc_step(θ, lj, lj_func, grad_func, H_func, ϵ, alg::HMC, momentum_sampler;
```

Comment:
Why is `momentum_sampler` not a function anymore? Changes on the compiler should not effect this, or?

Reply:
If I remember correctly, this was just me getting too `Function`-allergic. I will return it for now.